src/helpers/getDtsSnapshot.ts

```diff
@@ -24,7 +25,14 @@ export const getDtsSnapshot = (
     return scriptSnapshot;
   }
 
-  const classes = getClasses({ css, fileName, logger, options, processor });
+  const classes = getClasses({
+    css,
+    fileName,
+    logger,
+    options,
+    processor,
+    compilerOptions: compileOpts,
```

Comment:
Can we keep this as `compilerOptions` throughout all files? :)

Reply:
Sure, I will make commit tonight